languages/elixir/exercises/concept/anonymous-functions/.docs/introduction.md

```diff
@@ -0,0 +1,65 @@
+Functions are treated as first class citizens in Elixir. This means functions:
+
+- Can be assigned to variables.
+- Can be passed around like data as arguments and return values.
+- Can be created dynamically.
+
+```elxir
+varaible = fn param ->
+  param + 1
+end
+```
```

Comment:
I think these lines can all be removed, as they are duplicated below (and it makes more sense to have the example _after_ the description of anonymous functions).

```suggestion
```